### PR TITLE
Realtime record_detail!

### PR DESCRIPTION
**Pull request by iceb0y**

### CODE REVIEW

File: vj4/ui/pages/record_detail.page.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { NamedPage } from 'vj/misc/PageLoader';
+
+const page = new NamedPage('record_detail', async () => {
+  const SockJs = await System.import('sockjs-client');
+  const DiffDOM = await System.import('diff-dom');
+
+  const sock = new SockJs(window.location.pathname + '/conn');
```
Comment (iceb0y) [inline vj4/ui/pages/record_detail.page.js]: @breeswish How to do this properly?

File: vj4/ui/templates/record_detail_status.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<div class="section visible" id="status">
```
Comment (iceb0y) [inline vj4/ui/templates/record_detail_status.html]: I noticed that the original div doesn't contain the "visible" class, and someone adds it automatically. @breeswish

Comment (breeswish) [inline vj4/ui/templates/record_detail_status.html]: Yes. `visible` is added in `Entry.js`.

File: vj4/ui/pages/record_detail.page.js (hunk 1 above)
Comment (breeswish) [inline vj4/ui/pages/record_detail.page.js]: Better to define connection endpoint in a JavaScript variable in HTML `<script>` tag (e.g. like `Context`)

**Pull request merged**